Classes/Neos/Neos/Ui/Domain/Model/Feedback/Operations/RenderNode.php

```diff
@@ -0,0 +1,157 @@
+<?php
+namespace Neos\Neos\Ui\Domain\Model\Feedback\Operations;
+
+use Neos\Neos\Ui\Domain\Model\FeedbackInterface;
+use TYPO3\TYPO3CR\Domain\Model\NodeInterface;
+use TYPO3\Neos\View\TypoScriptView as FusionView;
+use TYPO3\Flow\Mvc\Controller\ControllerContext;
+
+class RenderNode implements FeedbackInterface
+{
+    /**
+     * @var NodeInterface
+     */
+    protected $node;
+
+    /**
+     * @var array
+     */
+    protected $referenceData;
+
+    /**
+     * @var string
+     */
+    protected $mode;
+
+    /**
+     * Set the node
+     *
+     * @param NodeInterface $node
+     * @return void
+     */
+    public function setNode(NodeInterface $node)
+    {
+        $this->node = $node;
+    }
+
+    /**
+     * Get the node
+     *
+     * @return NodeInterface
+     */
+    public function getNode()
+    {
+        return $this->node;
+    }
+
+    /**
+     * Set the referenceData
+     *
+     * @param string $referenceData
+     * @return void
+     */
+    public function setReferenceData(array $referenceData)
```

Comment:
instead of get/setReferenceData, I'd rather like to use "typoScriptPathToBeRendered" or so